ambry-clustermap/src/main/java/com/github/ambry/clustermap/CloudServiceClusterChangeHandler.java

```diff
@@ -95,9 +95,12 @@ public void registerClusterMapListener(ClusterMapChangeListener clusterMapChange
   }
 
   @Override
-  public void getReplicaIdsByStates(Map<ReplicaState, List<AmbryReplica>> replicasByState,
-      Set<ReplicaState> states, AmbryPartition partition) {
-    throw new UnsupportedOperationException("Get snapshot of replica state is temporarily not supported");
+  public void getReplicaIdsByStates(Map<ReplicaState, List<AmbryReplica>> replicasByState, Set<ReplicaState> states,
+      AmbryPartition partition) {
+    replicasByState.put(ReplicaState.LEADER,
+        Collections.singletonList(partitionToReplica.get(Long.toString(partition.getId()))));
+    replicasByState.put(ReplicaState.STANDBY, Collections.EMPTY_LIST);
```

Comment:
Use `Collections.emptyList()` to suppress generic type warning

Reply:
used. Thanks!